tests/drivers/test_cryomagnetics_4g.py

```diff
@@ -0,0 +1,177 @@
+import logging
+from unittest.mock import MagicMock, Mock, call, patch
+
+import pytest
+
+from qcodes.instrument_drivers.cryomagnetics.Cryomagnetics4G import (
```

Comment:
```suggestion
from qcodes.instrument_drivers.cryomagnetics import (
```
The teses should import from the public interface